storybook/src/stories/Global/TopBar.stories.tsx

```diff
@@ -42,8 +115,27 @@ export const _TopBar = () => {
       href: '#'
     }
   ])
+  const notificationsHistory = object("Notifications History", allNotifications);
 
   // userDrawerBespoke: See examples for implementation of this prop.
 
-  return <Container><TopBar {...{loggedInUser, userSubmenu, hasSearch, hasLogout, hasNotifications, logoutLink, searchPlaceholder, hasLanguage, hasCurrentUser}}/></Container>;
+  return (
+    <Container>
+      <TopBar {...{
+        loggedInUser,
+        userSubmenu,
+        hasSearch,
+        hasLogout,
+        hasNotifications,
+        logoutLink,
+        searchPlaceholder,
+        hasLanguage,
+        hasCurrentUser,
+        notificationsHistory
+        }
+        }
```

Comment:
Is this a linting thing? It's a bit confusing to have the opening `{` on the same lines but the closing `}` on separate lines.

Reply:
I guess it's at the same level because if it was on one line it would look like this.
I have updated to be in the same line that te last variable to avoid confusion.

{...{ lots,of, props}}